torchvision/models/_api.py

```diff
@@ -1,19 +1,52 @@
+from __future__ import annotations
+
 import importlib
 import inspect
 import sys
 from dataclasses import dataclass, fields
 from inspect import signature
 from types import ModuleType
-from typing import Any, Callable, cast, Dict, List, Mapping, Optional, TypeVar, Union
+from typing import Any, Callable, cast, Dict, List, Mapping, Optional, Tuple, TypeVar, Union
 
 from torch import nn
 
 from torchvision._utils import StrEnum
 
 from .._internally_replaced_utils import load_state_dict_from_url
 
-
-__all__ = ["WeightsEnum", "Weights", "get_model", "get_model_builder", "get_model_weights", "get_weight", "list_models"]
+__all__ = [
+    "TransformsFactory",
+    "WeightsEnum",
+    "Weights",
+    "get_model",
+    "get_model_builder",
+    "get_model_weights",
+    "get_weight",
+    "list_models",
+]
+
+
+@dataclass(init=False)
```

Comment:
[Docs](https://docs.python.org/3/library/dataclasses.html#module-contents) say

> If the class already defines __init__(), this parameter is ignored.

so we may not need it?

```suggestion
@dataclass
```

Reply:
Good catch!